manager/service.go

```diff
@@ -24,6 +26,10 @@ var (
 	ErrNotFound = errors.New("entity not found")
 )
 
+const (
```

Comment:
No need for these brackets when you have only one constant. these are used to group multiple consts.